### PR TITLE
Traverse the source zarr store copying the groups and attributes into…

### PR DESCRIPTION
**Pull request by flamingbear**
:wave: hey there.  I looked into this a bit for my work and wanted to leave this for someone in case they thought it was useful.  In my work, I'm still using v0.5 and running this basic command after the rechunk happens to pick up all of the missing group metadata. Hope this is helpful. If I get some time I will try to update with some tests if this seems useful.

### CODE REVIEW

**Comment by rabernat** [comment]
Welcome Mike and thanks for your contribution! I agree this seems useful.

I don't understand why the test workflow did not run on this PR. 🤔

**Comment by flamingbear** [comment]
CI didn't kick off, because actions weren't enabled on my fork. I pushed a space change to run them.

**Comment by rabernat** [comment]
Awesome!

If you can add a test for this, happy to merge it.

**Comment by flamingbear** [comment]
There's some CI permission stuff it looks like I'll have to dig into. Thanks.

**Comment by rabernat** [comment]
I'm guessing that some of the test failures we are seeing are due to a regression with Zarr.

**Comment by rabernat** [comment]
Perfect, CI is back in good shape. Now we just need a test that covers this new feature and we're good to go! 🚀

**Comment by flamingbear** [comment]
Appreciate it.  I will take a look and see if I can make sense of the tests.

**Comment by flamingbear** [comment]
@rabernat I hope this was you were looking for.  Hmm.  I thought I saw a bad coverage pop up. I didn't unit test, but added a second group test where it seemed reasonable to me.

**Review by rabernat — CHANGES_REQUESTED** [review]
Just one small request...

File: tests/test_rechunk.py [hunk 1]
```diff
@@ -488,6 +488,80 @@ def test_rechunk_group(tmp_path, executor, source_store, target_store, temp_stor
...
+    attr_values = [
+        ("/", "root description"),
+        ("foo", "foo group description"),
+        ("foo/a", "a array description"),
+        ("foo/bar", "bar group description"),
+        ("foo/bar/baz", "baz group description"),
+        ("foo/bar/baz/b", "b array description"),
+    ]
+    for attr_loc, attr_value in attr_values:
+        assert target_root[attr_loc].attrs["description"] == attr_value
```
[the remark this answers is not in the document]
Comment (flamingbear) [reply tests/test_rechunk.py:556]: I'm happy to do that, I just didn't want to mix up the previous test with my stuff, but yes, it's totally duplicative.

**Comment by rabernat** [comment]
I just released 0.5.1, which has this in it.

**Pull request merged**